src/AutoRoller.sol

```diff
@@ -665,6 +697,33 @@ contract AutoRoller is ERC4626 {
 
     /* ========== GENERAL UTILS ========== */
 
+    /// @dev Reinvests assets held by this contract into the Roller's liquidity strategy,
+    ///      thereby densifying each vault share
+    /// @param assets Amount of asset
+    function _densifyShares(uint256 assets) internal {
+        uint256 _pti    = pti;
+        bytes32 _poolId = poolId;
+        (uint256 ptReserves, uint256 targetReserves) = _getSpaceReserves();
+        (ERC20[] memory tokens, uint256[] memory balances, ) = balancerVault.getPoolTokens(_poolId);
+
+        uint256 targetForIssuance = _getTargetForIssuance(ptReserves, targetReserves, assets, adapter.scaleStored());
+
+        balances[1 - _pti] = assets - targetForIssuance;
+        if (assets - targetForIssuance > 0) {
+            balances[_pti] = divider.issue(address(adapter), maturity, targetForIssuance);
+        }
```

Comment:
I'm having trouble understanding why issuance lives within this `if` statement. Shouldn't `densifyShares` always be issuing?

I recommend adding a comment that answers the following question: what circumstances would cause `targetForIssuance` to be more than `assets`?